<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label1">
         <property name="text">
          <string>product Id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="product_Id"/>
       </item>
       <item>
        <widget class="QLabel" name="label1_2">
         <property name="text">
          <string>Product Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="product_name"/>
       </item>
       <item>
        <widget class="QLabel" name="label1_3">
         <property name="text">
          <string>description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="product_description"/>
       </item>
       <item>
        <widget class="QLabel" name="label1_4">
         <property name="text">
          <string>price</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="product_price"/>
       </item>
       <item>
        <widget class="QLabel" name="label1_5">
         <property name="text">
          <string>category</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="product_category"/>
       </item>
       <item>
        <widget class="QPushButton" name="createProduct">
         <property name="text">
          <string>Create/Update</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="editProduct">
         <property name="text">
          <string>Edit Product</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="productsTable">
         <column>
          <property name="text">
           <string>Proudct Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>category</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteProduct">
         <property name="text">
          <string>Delete Product</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClasses">
    <property name="title">
     <string>Classes</string>
    </property>
    <addaction name="actionClient"/>
    <addaction name="actionEmployee"/>
    <addaction name="actionPurchase"/>
   </widget>
   <addaction name="menuClasses"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClient">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionEmployee">
   <property name="text">
    <string>Employee</string>
   </property>
  </action>
  <action name="actionPurchase">
   <property name="text">
    <string>Purchase</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
